### PR TITLE
CRI-O: fix Type=notify and include iptables

### PR DESCRIPTION
**Pull request by giuseppe**
marked as WIP while waiting for @ashcrow tests results

### CODE REVIEW

File: cri-o-centos/config.json.template [hunk 1]
```diff
@@ -329,6 +329,17 @@
 		"mode=755"
 	    ]
 	},
+	{
```
Comment (ashcrow) [inline cri-o-centos/config.json.template]: 👍

File: cri-o-centos/Dockerfile [hunk 2]
```diff
@@ -14,7 +14,8 @@ LABEL com.redhat.component="cri-o" \
 COPY tmpfiles.template config.json.template service.template /exports/
 
 RUN yum-config-manager --nogpgcheck --add-repo https://cbs.centos.org/repos/virt7-docker-common-candidate/x86_64/os/ && \
-    yum install --nogpgcheck --setopt=tsflags=nodocs -y cri-o cri-o-cni && \
+    yum install --nogpgcheck --setopt=tsflags=nodocs -y iptables cri-o && \
```
Comment (ashcrow) [inline cri-o-centos/Dockerfile]: Why did you remove ```cri-o-cni```?

Comment (giuseppe) [inline cri-o-centos/Dockerfile]: it is called `containernetworking-cni-` on CentOS:

https://cbs.centos.org/repos/virt7-docker-common-candidate/x86_64/os/Packages/

It didn't fail before as we had no `rpm -V` and yum install is happy if any package is installed.  It is not needed anyway as it is a dependency of `cri-o`

**Comment by giuseppe** [comment]
@ashcrow if it works for you, could you r+ it? :-)

**Comment by ashcrow** [comment]
@rh-atomic-bot r+ 9760b71ec9ed62f0a3d3658c6b4770767a90dc89

**Comment by ashcrow** [comment]
@rh-atomic-bot try

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
Not sure why the bot wants to use commit 3 of 4 but I'll go with it.

@rh-atomic-bot r+ 31f3a44

**Comment by giuseppe** [comment]
@rh-atomic-bot retry
